### PR TITLE
Cleanup react routing and update upload page wording

### PR DESCRIPTION
**Pull request by drewbutlerbb4**
Cleanup react routing to make it more readable, as well as to set up two future PRs around an added 404 page for missed routes and reworking the server routing.

Also,
Resolves: #56

### CODE REVIEW

File: dashboard/origin-mlx/src/App.tsx [hunk 1]
```diff
@@ -79,398 +78,14 @@ function App() {
...
-                      )}
-                    />
-                  </Switch>
-                  { isAdmin &&
-                  <Route
-                    path="/experiments"
-                    render={({match, location}) =>
-                      <IframePage
-                        title="KFP Experiments"
-                        path={KFP + "/_/pipeline/?ns=mlx#/experiments" + window.location.pathname.substring(window.location.pathname.indexOf("/experiments")+12)}
```
Comment (yhwang) [inline dashboard/origin-mlx/src/App.tsx:441]: need to use `"/_/pipeline/?ns=mlx#/experiments"` for kfp v1.3. I also open an issue for this: #77

Comment (drewbutlerbb4) [reply dashboard/origin-mlx/src/App.tsx:441]: Thanks @yhwang, didn't see that you had updated that already.

**Review by yhwang — APPROVED** [review]
good for the condition aggregation!
/lgtm

**Comment by Tomcli** [comment]
@drewbutlerbb4 FYI I need to revert this PR because it broke the details page during login, can you open a new PR for this? Thanks.

**Comment by Tomcli** [comment]
you can reproduce the error when you run the quickstart.sh with your images.

**Pull request merged**